docs/release-notes.md

```diff
@@ -78,7 +74,11 @@ For full release details, refer to [Releases](https://github.com/n8n-io/n8n/rele
 View the [commits](https://github.com/n8n-io/n8n/compare/n8n@1.26.0...n8n@1.27.1){:target=_blank .external-link} for this version.<br />
 **Release date:** 2024-01-31
 
-This release contains node enhancements and bug fixes.
+This release contains new features, node enhancements, and bug fixes.
+
+/// warning | Breaking change
+This release removes `own` mode for self-hosted n8n. The remaining mode options are `main` and `queue`. Refer to [Queue mode](/hosting/scaling/queue-mode/) for information on configuring queue mode.
```

Comment:
We have removed `executions_process` fully so `main` isn't an option anymore either, The remaining setting is `execitons_mode` which is `regular` or `queue`